dll/win32/uxtheme/CMakeLists.txt

```
@@ -19,6 +19,11 @@ list(APPEND SOURCE
     uxthemep.h
     ${CMAKE_CURRENT_BINARY_DIR}/uxtheme_stubs.c)
 
+    if(DLL_EXPORT_VERSION GREATER_EQUAL 0x600)
+        list(APPEND SOURCE
+            pngsup.cpp)
+    endif()
+
```

Incorrect indentation:
```suggestion
if(DLL_EXPORT_VERSION GREATER_EQUAL 0x600)
    list(APPEND SOURCE
        pngsup.cpp)
endif()

```